.github/workflows/prod.yml

```diff
@@ -45,14 +45,15 @@ jobs:
 
       - name: Determine changes files
         run: |
+          PREVIOUS_TAG=$(git tag --sort version:refname | tail -n 2 | head -n 1)
           # Create a comma-separated list of changed files for use in build.gradle.kts
-          CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r -m $GITHUB_SHA | tr '\r\n' ',' | sed -e 's/,$//' | tr -d '"')
+          CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r -m $PREVIOUS_TAG $GITHUB_REF | tr '\r\n' ',' | sed -e 's/,$//' | tr -d '"')
```

Comment:
`PREVIOUS_TAG` er nest siste tag. `GITHUB_REF` er siste tag, som er en innebygget variabel i Github Actions.
Vi sammenliger disse for å se hvilke filer som har endret seg.